gossipd/gossip_wire.csv

```diff
@@ -24,6 +24,9 @@ gossipctl_init,,no_reconnect,bool
 # Gossipd->master, I am ready.
 gossipctl_init_reply,3100
 
+gossipctl_init,,lslen,u16
```

Comment:
Should this not be with the `gossipctl_init` fields *before* `gossipctl_init_reply`?  Clearer that way.